platform/ios/display_server_ios.mm

```diff
@@ -85,7 +92,19 @@
 		rendering_context = memnew(RenderingContextDriverVulkanIOS);
 	}
 #endif
-
+#ifdef METAL_ENABLED
+	if (rendering_driver == "metal") {
+		if (@available(iOS 14.0, *)) {
+			layer = [AppDelegate.viewController.godotView initializeRenderingForDriver:@"metal"];
+			wpd.metal.layer = (CAMetalLayer *)layer;
+			rendering_context = memnew(RenderingContextDriverMetal);
+		} else {
+			OS::get_singleton()->alert("Metal is only supported on iOS 14.0 and later.");
+			r_error = ERR_UNAVAILABLE;
+			return;
+		}
+	}
```

Comment:
Since we're making `metal` the first/preferred driver option for iOS, shouldn't we implement graceful fallback to Vulkan/MoltenVK for iOS < 14.0?